pkg/karmadactl/cmdinit/utils/format_test.go

```diff
@@ -0,0 +1,330 @@
+package utils
+
+import (
+	"net"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func stringInSlice(target string, str_array []string) bool {
```

Comment:
To unify the parameter naming style of the entire repo, it is advised to use camel case naming.

Reply:
Lint error occurs here.

Reply:
Updated. thanks